src/main/kotlin/blackjack/model/Hand.kt

```diff
@@ -0,0 +1,37 @@
+package blackjack.model
+
+class Hand {
+    val dealtCards: MutableList<Card> = mutableListOf()
```

Comment:
The Hand object looks great! It seems you have a strong ability to abstract concepts. 😊